renderer/pages/settings.tsx

```diff
@@ -1,84 +1,63 @@
 import { useEffect, useState } from "react";
 import Link from "next/link";
 import { shallow } from "zustand/shallow";
-import FolderSelector from "../components/FolderSelector";
-import { useMainStore } from "../store/mainStore";
 import { useThemeStore } from "../store/themeStore";
-import TrelloConnection from "../components/TrelloConnection";
-import Office365Connection from "../components/Office365Connection";
-import GoogleConnection from "../components/GoogleConnection";
-import clsx from "clsx";
-import TimetrackerWebsiteConnection from "../components/TimetrackerWebsiteConncetion";
-import BetaToggle from "../components/ui/BetaToggle";
-import JiraConnection from "../components/JiraConnection";
 import MenuItem from "../components/ui/MenuItem";
 import ButtonTransparent from "../components/ui/ButtonTransparent";
 import { ChevronLeftIcon } from "@heroicons/react/24/outline";
-
-enum SidebarNavItem {
-  Connections = "Connections",
-  ReportsFolder = "Reports folder",
-  Theme = "Theme",
-  VersionSelect = "Version",
-}
+import { SidebarNavItem, SETTING_SECTIONS } from "../helpers/contstants";
 
 const SettingsPage = () => {
   const [theme] = useThemeStore(
     (state) => [state.theme, state.setTheme],
-    shallow
+    shallow,
   );
   const [isOSDarkTheme, setIsOSDarkTheme] = useState(true);
   const [currentMenuItem, setCurrentMenuItem] = useState<SidebarNavItem>(
-    SidebarNavItem.Connections
+    SidebarNavItem.Connections,
   );
 
+  const settingSections = SETTING_SECTIONS[currentMenuItem];
+
+  const handleThemeChange = (e) =>
+    e.matches ? setIsOSDarkTheme(true) : setIsOSDarkTheme(false);
+
   useEffect(() => {
     const mediaQueryList = window.matchMedia("(prefers-color-scheme: dark)");
 
     mediaQueryList.addListener(handleThemeChange);
     setIsOSDarkTheme(mediaQueryList.matches);
 
-    const mode =
-      (theme.os && isOSDarkTheme) || (!theme.os && theme.custom === "dark")
-        ? "dark bg-dark-back"
-        : "light bg-grey-100";
-
-    document.body.className = mode;
+    document.body.className = (theme.os && isOSDarkTheme) || (!theme.os && theme.custom === "dark")
+      ? "dark bg-dark-back"
+      : "light bg-grey-100";
 
     return () => {
       mediaQueryList.removeListener(handleThemeChange);
     };
   }, [theme, isOSDarkTheme]);
 
-  function handleThemeChange(e) {
-    if (e.matches) {
-      setIsOSDarkTheme(true);
-    } else {
-      setIsOSDarkTheme(false);
-    }
-  }
-
-  const getSection = () => {
-    switch (currentMenuItem) {
-      case SidebarNavItem.Connections:
-      default:
-        return <ConnectionsSection />;
-
-      case SidebarNavItem.ReportsFolder:
-        return <ReportsFolderSection />;
-
-      case SidebarNavItem.Theme:
-        return <ThemeSection />;
-
-      case SidebarNavItem.VersionSelect:
-        return <VersionSection />;
-    }
-  };
+  const renderSidebarNavItem = () =>
+    Object.values(SidebarNavItem).map((value) => (
+      <li
```

Comment:
and, if I'm not mistaken, this variable should be in plural, like `renderSidebarNavItems`

Reply:
Yes, you're right.

Reply:
Resolved.